Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Multiplayer Samples Co-op Changelog
```

Comment:
should this changelog be udpated too to 2.5.0? It seems that it stopped at v1.9.0

Reply:
That's intended. Boss Room and the Utilities package are at different versions, but kept up to date separately per release.